PoGo.NecroBot.CLI/Config/Translations/translation.zh_TW.json

```diff
@@ -326,15 +326,15 @@
     },
     {
       "Key": "CurrentPotionInv",
-      "Value": "Potions: {0} | SuperPotions: {1} | HyperPotions: {2} | MaxPotions: {3}"
+      "Value": "恢復藥水: {0} | 超級恢復藥水: {1} | 超群恢復藥水: {2} | 完全恢復藥水: {3}"
```

Comment:
This is a problem. Because there is no official translation for items, there are a large variety of naming. And my version is out from this unofficial manual. http://forum.gamer.com.tw/G2.php?bsn=29659&parent=66&sn=19 But I think your recommendation is better.